app/lib/pre_assembly/digital_object.rb

```diff
@@ -46,9 +46,11 @@ def content_md_creation_style
         Cocina::Models::Vocab.image => :simple_image,
         Cocina::Models::Vocab.object => :file,
         Cocina::Models::Vocab.book => :simple_book,
-        Cocina::Models::Vocab.manuscript => :simple_book,
+        Cocina::Models::Vocab.document => :document,
         Cocina::Models::Vocab.map => :map,
-        Cocina::Models::Vocab.three_dimensional => :'3d'
+        Cocina::Models::Vocab.three_dimensional => :'3d',
+        Cocina::Models::Vocab.webarchive_seed => :webarchive_seed,
+        Cocina::Models::Vocab.media => :media
```

Comment:
Note that for this to work, we'd need to update the content-metadata creation gem to understand and produce the correct styles for webarchive_seed and media.  See https://github.com/sul-dlss/assembly-objectfile/blob/main/lib/assembly-objectfile/content_metadata.rb#L17

Up until now, media has a specialize way of making contentMetadata by inputing a specific CSV file.